### PR TITLE
RDAPI-29973 K8ssandra Layout Changes

### PR DESCRIPTION
**Pull request by vobrien-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

Initial layout changes for k8ssandra. Move old instructions to "classic" sections and create new Container sections

## Deploy preview link

http://pr-2663.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/apim_installation/apigtw_install/install_cassandra/

http://pr-2663.axway-open-docs.opendocs-builder.pcloud.axway.int/docs/cass_admin/

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 9b5b1bf |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-2663.axway-open-docs.opendocs-builder.pcloud.axway.int |

File: content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra/upgrade_cassandra_containers.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+"title": "Upgrade an Apache Cassandra in Containers",
```
Comment (andreamussap) [inline content/en/docs/apim_installation/apigw_upgrade/upgrade_cassandra/upgrade_cassandra_containers.md:2]: I've hidden this page. Discussing it with @pfoleyAx in Teams.

File: content/en/docs/cass_admin/admin_cassandra_classic/cassandra_bur.md [hunk 2]
```diff
@@ -86,8 +86,6 @@ The following table describes the parameters in the `apigw-backup-tool.ini` file
 | backup_root_dir            | Path to the backup directory                                            |
 | printcmd                   | Set to `true` to see the commands ran by the tool during the process          |
 | debug                      | Set to `true` to see debug logs                                               |
-| retry_index_count          | Set the number of attempts to restore a index                                 |
```
Comment (kmercierwhite) [inline content/en/docs/cass_admin/admin_cassandra_classic/cassandra_bur.md:89]: This needs to be kept, we added this in the aug23 release, so the k8ssandra branch might not have had it.

**Review by kmercierwhite — APPROVED** [review]

File: content/en/docs/cass_admin/admin_cassandra_classic/cassandra_bur.md (hunk 2 above)
Comment (andreamussap) [reply content/en/docs/cass_admin/admin_cassandra_classic/cassandra_bur.md:89]: done

**Pull request merged**